cmd/http-server/startcmd/start.go

```diff
@@ -654,11 +655,18 @@ func startHTTPServer(parameters *httpServerParameters) error {
 		return fmt.Errorf("failed to configure router: %w", err)
 	}
 
+	handler := cors.New(
+		cors.Options{
+			AllowedMethods: []string{http.MethodGet, http.MethodPost, http.MethodDelete, http.MethodHead},
```

Comment:
Delete? Head? Do we need those?

Reply:
deleted 'delete' & 'head', we aren't using both for now.